packages/components/src/DateTimePickers/InputDateRangePicker/InputDateRangePicker.scss

```diff
@@ -0,0 +1,31 @@
+$tc-datepicker-width: 31rem !default;
+$tc-datepicker-background: $white !default;
+$tc-datepicker-box-shadow: $shadow-default !default;
+$tc-datepicker-z-index: $zindex-dropdown !default;
+$tc-range-picker-arrow-height: 3rem !default;
+
+.popper {
+	width: $tc-datepicker-width;
+	background: $tc-datepicker-background;
+	box-shadow: $tc-datepicker-box-shadow;
+	z-index: $tc-datepicker-z-index;
+}
+.date-picker {
+	width: fit-content;
+	display: flex;
+	align-items: flex-end;
+}
```

Comment:
Please don't duplicate that for each picker. Extract that somewhere common and import this common style in each picker to set the popper classname